src/components/BuildDetails.tsx

```diff
@@ -18,9 +18,10 @@ const BuildDetails: React.FunctionComponent = () => {
   }
 
   const loadingAnimation = selectedBuild.isRunning && <LinearProgress />;
+  const locatorBuildDetails = "build-details";
```

Comment:
could you move those kind of locators into `constants/help.ts`?

Reply:
Done.